diff --git a/openssh.rb b/openssh.rb
index 260ee93..c07aacf 100644
--- a/openssh.rb
+++ b/openssh.rb
@@ -16,6 +16,7 @@ class Openssh < Formula
   # Please don't resubmit the keychain patch option. It will never be accepted.
   # https://github.com/Homebrew/homebrew-dupes/pull/482#issuecomment-118994372
   option "with-libressl", "Build with LibreSSL instead of OpenSSL"
+  option "with-ssh1", "Build with ssh1 support"
 
   depends_on "openssl" => :recommended
   depends_on "libressl" => :optional
@@ -43,6 +44,12 @@ class Openssh < Formula
       url "https://raw.githubusercontent.com/Homebrew/patches/d8b2d8c2/OpenSSH/launchd.patch"
       sha256 "df61404042385f2491dd7389c83c3ae827bf3997b1640252b018f9230eab3db3"
     end
+
+    # Patch for macOS
+    patch do
+      url "https://raw.githubusercontent.com/boltomli/MyMacScripts/master/homebrew/openssh.macOS.patch"
+      sha256 "546464d165dfeb1ec261dd81447aeee06ac875960f32031f623cb49aaa1074e3"
+    end
   end
 
   def install
@@ -65,6 +72,8 @@ class Openssh < Formula
 
     args << "--with-ldns" if build.with? "ldns"
 
+    args << "--with-ssh1" if build.with? "ssh1"
+
     system "./configure", *args
     system "make"
     system "make", "install"
