--- /Users/songl/openssh-7.1p2/openbsd-compat/setproctitle.c	2016-01-14 09:10:45.000000000 +0800
+++ /Users/songl/openssh-7.2p2/openbsd-compat/setproctitle.c	2016-07-21 13:22:47.000000000 +0800
@@ -149,8 +149,8 @@
 	va_end(ap);
 	if (r == -1 || (size_t)r >= sizeof(buf) - len)
 		return;
-	strnvis(ptitle, buf, sizeof(ptitle),
-	    VIS_CSTYLE|VIS_NL|VIS_TAB|VIS_OCTAL);
+//	strnvis(ptitle, buf, sizeof(ptitle),
+//	    VIS_CSTYLE|VIS_NL|VIS_TAB|VIS_OCTAL);
 
 #if SPT_TYPE == SPT_PSTAT
 	pst.pst_command = ptitle;
